### ISSUE TITLE
Can't fetch from MELPA

### ISSUE THREAD

**Issue by geokon-gh** [stardiviner/ob-clojure-literate#4]
On: `GNU Emacs 25.2.2`
Running:

``` elisp
(use-package ob-clojure-literate
  :ensure t
  :after org
  :init
  (setq ob-clojure-literate-auto-jackin-p t)
  (add-hook 'org-mode-hook #'ob-clojure-literate-mode))
```

Yields
```
Debugger entered--Lisp error: (error "Package ‘ob-clojure-literate-’ is unavailable")
  signal(error ("Package ‘ob-clojure-literate-’ is unavailable"))
  error("Package `%s-%s' is unavailable" ob-clojure-literate "")
  package-compute-transaction(nil ((ob-clojure-literate)))
  package-install(ob-clojure-literate)
[...]
```

This seems to mention someone else having the same issue in the recent past

https://www.reddit.com/r/orgmode/comments/akr4p4/latest_orgmode_package_updates_errors_on/?ref=readnext

**Comment by stardiviner** [comment]
Sorry for late response. Because I'm watch many repos, so I have not noticed this issue.

This package is alreay move to org-contrib a few monthes ago. I used to add a declaration link https://github.com/stardiviner/ob-clojure-literate#deprecated in README. Also, because of `CIDER` currently use `sesman` to manage REPL sessions, so I migrated `ob-clojure-literate` some functions to `ob-clojure` already. So you don't need the `ob-clojure-literate` now, if you're using latest version of Org (maybe `org 9.2` at least? I have not tested).

And thanks for using my package and filling an issue. I will archive this repository now in Github repo settings.

I you have any question, ask as you want.
